fabric-biome-api-v1/src/testmod/java/net/fabricmc/fabric/test/biome/FabricBiomeTest.java

```diff
@@ -86,32 +86,38 @@ public void onInitialize() {
 
 		// TESTING HINT: to get to the end:
 		// /execute in minecraft:the_end run tp @s 0 90 0
-		TheEndBiomes.addHighlandsBiome(TEST_END_HIGHLANDS, 5.0);
-		TheEndBiomes.addMidlandsBiome(TEST_END_HIGHLANDS, TEST_END_MIDLANDS, 1.0);
+		org.quiltmc.qsl.worldgen.biome.api.TheEndBiomes.addHighlandsBiome(TEST_END_HIGHLANDS, 5.0);
+		org.quiltmc.qsl.worldgen.biome.api.TheEndBiomes.addMidlandsBiome(TEST_END_HIGHLANDS, TEST_END_MIDLANDS, 1.0);
```

Comment:
Ideally the testmod should still use the fabric API stuff, not QSL - this is the testmod for QFAPI, not QSL, after all